general_itests/fake_etc_paasta/volumes.json

```diff
@@ -0,0 +1,9 @@
+{
+    "volumes": [
+        {
+          "hostPath": "/nail/etc/services",
```

Comment:
For itests can you use something less yelpy to make it more obvious that we are not accidentally depending on real resources?